GitCommands/Settings/AppSettings.cs

```diff
@@ -1313,22 +1313,19 @@ public static bool ShowCommitBodyInRevisionGrid
             set => SetBool("ShowCommitBodyInRevisionGrid", value);
         }
 
-        /// <summary>Gets or sets the path to the git application executable.</summary>
-        public static string GitBinDir
+        /// <summary>Gets or sets the path to the GNU tools.</summary>
+        public static string GnuToolsDir
         {
-            get => GetString("gitbindir", "");
+            get => GetString(nameof(GnuToolsDir), defaultValue: null) ?? GetString("gitbindir", "");
             set
             {
-                var temp = value.EnsureTrailingPathSeparator();
-                SetString("gitbindir", temp);
-
-                ////if (string.IsNullOrEmpty(_gitBinDir))
-                ////   return;
-                ////
-                ////var path =
-                ////   Environment.GetEnvironmentVariable("path", EnvironmentVariableTarget.Process) + ";" +
-                ////   _gitBinDir;
-                ////Environment.SetEnvironmentVariable("path", path, EnvironmentVariableTarget.Process);
+                string gnuToolsDir = value;
+                if (!string.IsNullOrWhiteSpace(gnuToolsDir))
+                {
+                    gnuToolsDir = gnuToolsDir.EnsureTrailingPathSeparator();
+                }
+
+                SetString(nameof(GnuToolsDir), gnuToolsDir);
```

Comment:
Why not keep the existing name?
This will require users to configure this again

Reply:
> This will require users to configure this again

I think this takes care of it:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/48afcbbb-704c-4e07-8786-6f42e75c6d3c)

Though we should add a comment explaining this takes care of backwards compatibility.